frontend/src/core/comments/components/Comment.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+
+import * as React from 'react';
+import ReactTimeAgo from 'react-time-ago';
+
+import './Comment.css';
+
+import type { TranslationComment } from 'core/api';
+
+import { UserAvatar } from 'core/user'
+
+
+type Props = {|
+    comment: TranslationComment,
+|};
+
+
+export default function Comment(props: Props) {
+    const { comment } = props;
+
+    if (!comment) {
+        return null;
+    }
+
+    return <li className='history-comment'>
+        <UserAvatar
+            user={ comment.author }
+            username={ comment.username }
+            title=''
+            imageUrl={ comment.userGravatarUrlSmall }
+        />
+        <div className='container'>
+            <div className='content'>
+                <a
+                    href={ `/contributors/${comment.username}` }
+                    target='_blank'
+                    rel='noopener noreferrer'
+                    onClick={ (e: SyntheticMouseEvent<>) => e.stopPropagation() }
+                >
+                    { comment.author }
+                </a>
+                <p>
+                    { comment.content }
+                </p>
+            </div>
+            <div className='info'>
+                <ReactTimeAgo
+                    dir='ltr'
+                    date={ new Date(comment.dateIso) }
+                    title={ `${comment.createdAt} UTC` }
+                />
+                { '\u2022' }
```

Comment:
These are purely decorative, so I suggest we handle them with [CSS psuedo-classes](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes) like for the Translation metadata.